.travis.yml

```diff
@@ -1,4 +1,10 @@
 services:
   - docker
+language: python
+python:
+  - "3.7"
+cache: pip
 script:
   - ./test.sh
+  - pip install -r requirements.txt
+  - ./test_cwl.sh
```

Comment:
```suggestion
```
Prefer to have everything in test.sh... but test.sh could in turn call ./test_cwl.sh.